distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email/Email_Attachment.enso

```diff
@@ -2,14 +2,16 @@ import project.Data.Text.Text
 import project.Errors.Common.Missing_Argument
 import project.Metadata.Display
 import project.Metadata.Widget
+import project.Network.URI.URI
 import project.System.File.File
 from project.Metadata.Choice import Option
 from project.Metadata.Widget import Text_Input, Vector_Editor, Single_Choice
+import project.Enso_Cloud.Enso_File.Enso_File
 
 ## The attachment of an email.
 type Email_Attachment
     ## A file attachment.
-    File file:File=(Missing_Argument.throw "file") name:Text=""
+    File file:Text|URI|File|Enso_File=(Missing_Argument.throw "file") name:Text=""
```

Comment:
This type checking will mean you can't use `S3_File` or `OneDrive_File` - I'd suggest removing it.
Alternatively, could use `File_Like` I think for `File|Enso_File` and that should work.